src/sagas/application.js

```diff
@@ -172,6 +172,39 @@ function* loginUserWorker({ emailAddress, password }) {
   }
 }
 
+function* logoutUserWorker() {
+  try {
+    // first, check to see if we are logged in
+    const loggedIn = yield select(isLoggedIn);
+
+    if (loggedIn) {
+      localStorage.setItem('accessToken', null);
```

Comment:
Probably want `localStorage.removeItem` here - [docs](https://developer.mozilla.org/en-US/docs/Web/API/Storage/removeItem)

Reply:
Refactored to `localStorage.removeItem()`.  I had read somewhere that `localStorage.setItem()` is better for performance than deleting and creating, but I can't find the reference; in this case I doubt there it makes a difference one way or the other.